.github/workflows/quic_matrix.json

```diff
@@ -1,13 +1,9 @@
 [
-    { "env": "azure-ex", "os": "ubuntu-20.04", "arch": "x64", "tls": "openssl",  "io": "epoll" },
+    { "env": "azure", "os": "ubuntu-20.04", "arch": "x64", "tls": "openssl",  "io": "epoll" },
     { "env": "azure", "os": "windows-2022", "arch": "x64", "tls": "schannel", "io": "iocp" },
     { "env": "azure", "os": "windows-2022", "arch": "x64", "tls": "schannel", "io": "xdp" },
     { "env": "azure", "os": "windows-2022", "arch": "x64", "tls": "schannel", "io": "rio" },
     { "env": "azure", "os": "windows-2022", "arch": "x64", "tls": "schannel", "io": "wsk" },
-    { "env": "azure", "os": "windows-2025", "arch": "x64", "tls": "schannel", "io": "iocp" },
```

Comment:
Why remove these?

Reply:
Due to the S360 issue, we had to delete the F4-ubuntu-07 VM. 

If we leave these, then all future runs will hang.

We can add them back once we get permission to create WS2025 machines dynamically, or we settle on a stable static WS2025 VMs on Azure.